packages/client-search/src/types/SearchResponse.ts

```diff
@@ -28,6 +28,11 @@ export type SearchResponse<TObject = {}> = {
    */
   nbHits: number;
 
+  /**
+   * Subset of hits selected when relevancyStrictness is applied.
+   */
+  nbSortedHits: number;
```

Comment:
Sorry, I didn't see it in my first review but, `nbSortedHits` is optional and will only be present when querying VRs.
Is that good with your current implem?

Reply:
ah yes thanks for catching, it's meant to have a question mark!

```suggestion
  nbSortedHits?: number;
```